core/tauri-runtime/src/lib.rs

```diff
@@ -528,6 +529,21 @@ pub trait Dispatch<T: UserEvent>: Debug + Clone + Send + Sync + Sized + 'static
   /// Hides the window.
   fn hide(&self) -> Result<()>;
 
+  /// Shows the application on MacOS (independent of current window).
```

Comment:
I think some more explanations would be helpful here, what does hiding the app mean in detail? Does it affect the dock icon? Does it affect Mission Control? Or the app quick switcher?

Reply:
I'd love to, but the only way to know would be experimental. Apple is deliberately vague about this, probably because things like dock and mission control are subject to changes across MacOS versions. Being more detailed than Apple's own docs won't be much help. Here's an example of what we call `show_application`:

https://developer.apple.com/documentation/appkit/nsapplication/1428761-unhide?language=objc